lib/Model/Importer/DebugInfo/PDBImporter.cpp

```diff
@@ -706,15 +671,17 @@ Error PDBImporterTypeVisitor::visitKnownRecord(CVType &Record,
   bool WasReferenced = ForwardReferencedTypes.count(CurrentTypeIndex) != 0;
   if (InProgressMemberTypes.count(FieldsTypeIndex) != 0) {
     model::StructDefinition *Struct = nullptr;
-    auto NewType = makeTypeDefinition<model::StructDefinition>();
+    auto NewDefinition = makeTypeDefinition<model::StructDefinition>();
     if (not WasReferenced) {
-      NewType->OriginalName() = Class.getName();
-      auto NewStruct = cast<model::StructDefinition>(NewType.get());
-      NewStruct->Size() = Class.getSize();
-      Struct = NewStruct;
+      NewDefinition->OriginalName() = Class.getName();
+      auto &NewStruct = llvm::cast<model::StructDefinition>(*NewDefinition);
+      NewStruct.Size() = Class.getSize();
+      Struct = &NewStruct;
     } else {
       TypeIndex ForwardRef = ForwardReferencedTypes[CurrentTypeIndex];
-      Struct = cast<model::StructDefinition>(ProcessedTypes[ForwardRef].get());
+
+      using StructDefinition = model::StructDefinition;
+      Struct = cast<StructDefinition>(&ProcessedTypes[ForwardRef]->asStruct());
```

Comment:
I guess the cast is superfluous.

Reply:
Ah, it looks like one of the automated replacements. Thanks for noticing.

Reply:
Try `git grep ''cast<.*Definition>.*->as` or something like that, maybe we can catch some more.

Reply:
Done.